### PR TITLE
[[ Bug 22808 ]] Use default browser to show OAuth2 dialog on Linux

### PR DESCRIPTION
**Pull request by livecodepanos**
This patch uses the default browser instead of the browser widget to show OAuth2 dialog on Linux.

### CODE REVIEW

File: extensions/script-libraries/oauth2/oauth2.livecodescript [hunk 1]
```diff
@@ -1,4 +1,4 @@
-﻿script "com.livecode.library.oauth2"
+script "com.livecode.library.oauth2"
```
Comment (runrevmark) [inline extensions/script-libraries/oauth2/oauth2.livecodescript]: What's the reason for the change to the first line? (My guess is that a BOM has been added or removed)

**Review by runrevmark — CHANGES_REQUESTED** [review]
PR title / commit message title should have prefix `[[ Bug 22808 ]]`

File: extensions/script-libraries/oauth2/oauth2.livecodescript (hunk 1 above)
Comment (livecodepanos) [reply extensions/script-libraries/oauth2/oauth2.livecodescript]: Yeap, it seems it you edit a `.livecodescript` file in the IDE it removes the BOM.

I have fixed it now

**Comment by FourthWorld** [comment]
I very much appreciate Panos jumping on this several months ago, but progress seems to have stalled out on the Vulcan test. Is there anything I might be able to help with to move this along?

**Review by runrevmark — APPROVED** [review]

**Comment by runrevmark** [comment]
@livecode-vulcan review ok 47048e1

**Comment by livecode-vulcan** [comment]
💙 review by @runrevmark ok 47048e11255d0d5ee1df2d7251e858785f8ac943

**Comment by livecode-vulcan** [comment]
😎 test success 47048e11255d0d5ee1df2d7251e858785f8ac943

* try-community-armv6-android-sdk26_ndk16r15: [success](https://vulcan.livecode.com/builders/try-community-armv6-android-sdk26_ndk16r15/builds/984)
* try-community-armv7-android-ndk16r15: [success](https://vulcan.livecode.com/builders/try-community-armv7-android-ndk16r15/builds/392)
* try-community-arm64-android-ndk16r15: [success](https://vulcan.livecode.com/builders/try-community-arm64-android-ndk16r15/builds/391)
* try-community-x86-android-ndk16r15: [success](https://vulcan.livecode.com/builders/try-community-x86-android-ndk16r15/builds/390)
* try-community-x86_64-android-ndk16r15: [success](https://vulcan.livecode.com/builders/try-community-x86_64-android-ndk16r15/builds/391)
* try-community-js-emscripten-sdk1.35: [success](https://vulcan.livecode.com/builders/try-community-js-emscripten-sdk1.35/builds/2509)
* try-community-universal-ios-iphoneos14.4: [success](https://vulcan.livecode.com/builders/try-community-universal-ios-iphoneos14.4/builds/38)
* try-community-universal-ios-iphoneos14.1: [success](https://vulcan.livecode.com/builders/try-community-universal-ios-iphoneos14.1/builds/87)
* try-community-universal-ios-iphonesimulator14.4: [success](https://vulcan.livecode.com/builders/try-community-universal-ios-iphonesimulator14.4/builds/38)
* try-community-universal-ios-iphonesimulator14.1: [success](https://vulcan.livecode.com/builders/try-community-universal-ios-iphonesimulator14.1/builds/85)
* try-community-universal-mac-macosx10.9: [success](https://vulcan.livecode.com/builders/try-community-universal-mac-macosx10.9/builds/2536)
* try-community-x86-linux-debian8: [success](https://vulcan.livecode.com/builders/try-community-x86-linux-debian8/builds/2525)
* try-community-x86_64-linux-debian8: [success](https://vulcan.livecode.com/builders/try-community-x86_64-linux-debian8/builds/2523)
* try-community-x86-win32: [success](https://vulcan.livecode.com/builders/try-community-x86-win32/builds/2560)
* try-community-x86_64-win32: [success](https://vulcan.livecode.com/builders/try-community-x86_64-win32/builds/2496)
